### PR TITLE
removing reference to APAC DC

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Description
As per the TC meeting, APAC DC will be decommissioned on **June 12**, thus need to remove any reference in the docs

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/dev/cli/sauce-connect-proxy.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ You can view the entire list of CLI options by running the `--help` flag.
 
 <p><small>| OPTIONAL | STRING |  <span className="sauceRed">4.7.x</span> <span className="sauceRed">4.8.x</span>| </small></p>
 
-**Description**: Sets your Sauce Labs [data center endpoint](/basics/data-center-endpoints/#data-center-endpoints) (e.g., `us-west`, `eu-central`, `apac-southeast`).<br/>
+**Description**: Sets your Sauce Labs [data center endpoint](/basics/data-center-endpoints/#data-center-endpoints) (e.g., `us-west`, `eu-central`).<br/>
```
Comment (LoryApiFortress) [inline docs/dev/cli/sauce-connect-proxy.md:66]:
```suggestion
**Description**: Sets your Sauce Labs [data center endpoint](/basics/data-center-endpoints/#data-center-endpoints) (for example., `us-west`, `eu-central`).<br/>
```

**Review by aliciakrebs — APPROVED** [review]
Approved. NOTE: Hold merge until date is confirmed.

**Comment by LoryApiFortress** [comment]
Converted to Draft for the time being so that the Merge button is disabled to avoid accidentally merged

**Comment by lysannep** [comment]
@aliciakrebs @LoryApiFortress Should the [Early Access Australian Data Center](https://docs.saucelabs.com/basics/data-center-endpoints/aust-early-access/) page be removed as well?

**Comment by LoryApiFortress** [comment]
@lysannep, any reference to APAC and Headless should be removed from the docs, but I did this only for the docs I'm working on. I don't want to touch something that is not "mine". Headless is scheduled to be decommissioned tomorrow, Friday the 12th, and APAC has been pushed out to **June** 12.

File: docs/secure-connections/sauce-connect/system-requirements.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Enable your Mac to allow apps from the App Store and identified developers.
 
 1. Go to **System Preferences** > **Security & Privacy** > **General**.
 2. Under **Allow apps downloaded from**, select the option **App Store and identified developers**.
-:::
+   :::
```
Comment (lysannep) [inline docs/secure-connections/sauce-connect/system-requirements.md:77]:
```suggestion
:::
```

**Review by lysannep — APPROVED** [review]

**Comment by lysannep** [comment]
This PR should be merged with PR https://github.com/saucelabs/sauce-docs/pull/2002.

**Comment by aliciakrebs** [comment]
Closed with this PR: https://github.com/saucelabs/sauce-docs/pull/2002

**Pull request closed without merging**